docs/reference/master-configuration-reference.md

```diff
@@ -128,5 +128,64 @@ Default: 1 hour.
 Example:
 
 ```yaml
-resyncPeriod=2h
+resyncPeriod: 2h
+```
+
+### leaderElection
+
+The `leaderElection` option exposes configuration to tweak leader election.
```

Comment:
I'd suggest to reword this, e.g.

```suggestion
The `leaderElection` section exposes configuration to tweak leader election.
```

or 

```suggestion
The `leaderElection` options expose configuration to tweak leader election.
```

Reply:
fixed